static/js/settings_ui.js

```diff
@@ -13,6 +13,26 @@ exports.strings = {
     saving: i18n.t("Saving"),
 };
 
+exports.save_organization_settings = function (data, save_button) {
```

Comment:
Instead of passing the URL in next commit, we could have changed it here. And we can rename this function to, `save_subsection_settings`? Thoughts?